src/components/WebexRemoteMedia/WebexRemoteMedia.scss

```diff
@@ -4,9 +4,20 @@
   align-items: center;
   height: 100%;
   width: 100%;
-}
 
-.remote-media video {
-  width: 100%;
-  height: 100%;
+  span {
```

Comment:
Can you wrap this within `.md-badge` to make sure that is only affecting Momentum components?